### PR TITLE
Build prebid.js with specified adapters

### PR DESCRIPTION
**Pull request by protonate**
An `adapters` section in package.json allows for specifying which adapters to build with. This makes it easy to optimize the prebid file size by only including adapters that you need. Adapter names can be added and removed from the `adapters` array and only those adapter code files will be included in the build. If no adapters are found or no `adapters` section exists, then all adapters will be included. Adapter names must match the adapter file names.

review notes

* renamed file -> `adapterLoader.js`
* move version header task after uglify

updates

* split strings to array in favor of spread operator
* update README, add devpack task to watch and build

### CODE REVIEW

File: webpack.conf.js [hunk 1]
```diff
@@ -22,6 +22,11 @@ module.exports = {
       {
         test: /\.json$/,
         loader: 'json'
+      },
+      {
+        test: /adaptermanager.js/,
+        include: /(src)/,
+        loader: 'adapter-loader'
```
Comment (mkendall07) [inline webpack.conf.js]: let's use camelCase file names

**Comment by protonate** [comment]
@mkendall07 review notes addressed.

File: loaders/adapterLoader.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    return inserts.map((adapter) => {
+      return `var ${adapterName(adapter)} = require('./adapters/${adapter}.js');\n` +
+        `exports.registerBidAdapter(new ${adapterName(adapter)}` +
+        `${useCreateNew(adapter)}(), '${adapter}');\n`;
+    }).join('');
+  }
+};
+
+function adapterName(adapter) {
+  let result = [...adapter];
```
Comment (mkendall07) [inline loaders/adapterLoader.js]: @protonate  this line was causing trouble with node < 5.x.x. Please either replace it or put a dev note that the new build requires node > 5.x.x  as I'm sure someone else might also have this issue.

**Comment by protonate** [comment]
@mkendall07 updated with removal of spread operator and changes to README.

**Comment by mkendall07** [comment]
Thanks @protonate

**Pull request merged**